_posts/2018-06-18-configuration.md

```diff
@@ -0,0 +1,160 @@
+---
+author: Sean Callan
+author_link: https://github.com/doomspork
+categories: general
+date: 2018-06-18
+layout: post
+title:  Confusing Configuration
```

Comment:
Probably needs a better title.

Reply:
How about something along the lines of "Configuration Demystified" or even "Best Practices"?